[2024-11-09T18:36:22.110+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-09T18:36:22.139+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-09T18:36:10.046643+00:00 [queued]>
[2024-11-09T18:36:22.149+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-09T18:36:10.046643+00:00 [queued]>
[2024-11-09T18:36:22.150+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-09T18:36:22.165+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_csv> on 2024-11-09 18:36:10.046643+00:00
[2024-11-09T18:36:22.170+0000] {standard_task_runner.py:64} INFO - Started process 217 to run task
[2024-11-09T18:36:22.174+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_csv', 'manual__2024-11-09T18:36:10.046643+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmphn434a5j']
[2024-11-09T18:36:22.176+0000] {standard_task_runner.py:91} INFO - Job 401: Subtask fetch_data_from_csv
[2024-11-09T18:36:22.224+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-09T18:36:10.046643+00:00 [running]> on host b92a37539cfa
[2024-11-09T18:36:22.326+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T18:36:10.046643+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-09T18:36:10.046643+00:00'
[2024-11-09T18:36:22.327+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-09T18:36:22.328+0000] {logging_mixin.py:188} INFO - Error reading CSV file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.csv'
[2024-11-09T18:36:22.329+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-09T18:36:22.330+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-09T18:36:22.342+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_csv, run_id=manual__2024-11-09T18:36:10.046643+00:00, execution_date=20241109T183610, start_date=20241109T183622, end_date=20241109T183622
[2024-11-09T18:36:22.387+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-09T18:36:22.411+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-09T18:36:22.414+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
